qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -4,277 +4,27 @@
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
-from os import path, makedirs
-from shutil import copy2
-from numpy import array, degrees
-from mantid.api import AnalysisDataService as Ads
-from mantid.kernel import logger, UnitParams
-from mantid.simpleapi import PDCalibration, DeleteWorkspace, DiffractionFocussing, CreateDetectorTable,\
-    CreateEmptyTableWorkspace, NormaliseByCurrent, ConvertUnits, Load, SaveNexus, ApplyDiffCal
+from mantid.simpleapi import Load, AnalysisDataService as ADS, logger
 import Engineering.EnggUtils as EnggUtils
-from mantidqtinterfaces.Engineering.gui.engineering_diffraction.settings.settings_helper import get_setting
-from mantidqtinterfaces.Engineering.gui.engineering_diffraction.settings.settings_presenter import CALIB_FOLDER
 from mantidqtinterfaces.Engineering.gui.engineering_diffraction.tabs.common import output_settings
-from Engineering.common import path_handling
-
-DIFF_CONSTS_TABLE_NAME = "diffractometer_consts_table"
+from mantidqtinterfaces.Engineering.gui.engineering_diffraction.settings.settings_helper import get_setting
```

Comment:
This comment doesn't specifically relate to this line but just needed to attach it somewhere - might be worth adding a comment in this unit somewhere to explain the model shares code with the EnginX script unit that's used for auto reduction. The reader might work that out by following the imports but would make it quicker to understand